Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.cpp

```diff
@@ -603,7 +603,6 @@ namespace AzToolsFramework
                 clonedFilter->AddFilter(FilterConstType(customTypeFilter));
 
                 clonedFilter->SetFilterPropagation(AssetBrowserEntryFilter::PropagateDirection::Down);
-                m_assetFilterModel->SetFilter(FilterConstType(clonedFilter));
```

Comment:
This line is what caused the search crash

Reply:
I see the same call at the line N 609, I wonder what was the error, double call, too early call?
The removed line looks similar to the "else {}" block where SetFilter follow SetFilterPropgation, if the bug is about double call then maybe we should remove the one at 609. Anyway this is minor, I just curious what the problems was.

Reply:
For me it works with the first call commented out and with the second call commented out.